ui/apps/platform/cypress/integration/vulnerabilities/vulnerabilities.helpers.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Get the count of each severity label, returns an object with the count of each severity label.
+ * If a label is hidden, the count will be null.
+ *
+ * @param labelParentElement The parent element of the severity labels
+ * @returns The count of each severity label
+ */
+export function getSeverityLabelCounts(labelParentElement: HTMLElement) {
+    const labelValues = Array.prototype.slice
```

Comment:
What do you think about using `Array.from()` instead of `Array.prototype.slice.call()` to convert the NodeList to an array? The former seems less verbose and more intuitive.

Reply:
Ah good idea! An old habit I need to de-internalize 😄